<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyboardConfig</class>
 <widget class="QWidget" name="KeyboardConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Cursor flash time:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;b&gt;Keyboard&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="keyboardBeep">
     <property name="text">
      <string>Beep when there is an error of keyboard input</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="cursorFlashTime">
     <property name="suffix">
      <string> ms</string>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Character Repeat</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="6" column="0" colspan="4">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="2" column="2">
       <widget class="QSlider" name="keyboardInterval">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>210</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Long</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSlider" name="keyboardDelay">
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>1100</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
        <property name="pageStep">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Repeat delay:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Short</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Repeat interval:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="4">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Type in the following box to test your keyboard settings</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Short</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Long</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="keyboardNumLock">
     <property name="text">
      <string>Turn on NumLock after login</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>keyboardDelay</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_16</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>keyboardInterval</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_17</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
